.github/workflows/my-ci-pipeline.yml

```diff
@@ -38,6 +38,8 @@ jobs:
     run: docker build --target production --tag kelliep/my-todo-app .
   - name: docker push
     run: docker push kelliep/my-todo-app:latest
+  - name: deploy to Azure
+    run: curl -dH -X POST ${{ secrets.WEBHOOK }}
```

Comment:
One nice addition here can be adding the [--fail](https://curl.se/docs/manpage.html#-f) flag, as otherwise curl reports success as long as a successful connection was made, even if the response reports a failure (e.g. a status code > 400) which can lead to confusing pipeline results